ui_framework/doc_site/src/views/action_item/action_items_in_menu.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import {
+  KuiActionItem
+} from '../../../../components';
+
+export default () => (
+<div className="kuiMenu kuiMenu--contained">
```

Comment:
We should wait until https://github.com/elastic/kibana/pull/12135 is merged, and then use `<KuiMenu>` and `<KuiMenuItem>` here.